### ISSUE TITLE
Include a copyable line that I can add to my requirements.txt on the pypi home page for each library

### ISSUE THREAD

**Issue by michaelwiles** [pypa/warehouse#8019]
**What's the problem this feature will solve?**
Looking to increase efficiency and convenience

**Describe the solution you'd like**
So on the pypi home page for each library there is a pip install xxx line that you can copy. 

I'm requesting that a requirements.txt entry for the latest version of the library is also included here. So xxx==<LATEST_VERSION> that I can just copy so that I can paste it in to my requirements.txt. Not entirely sure how to present it on the pypi front page but this would be very helpful and not a lot of work to do.

**Comment by bpabel** [comment]
The [Packing User Guide](https://packaging.python.org/tutorials/managing-dependencies/) doesn't even cover requirements.txt files, so I'm assuming their use is de facto deprecated in favor of more modern tools like `poetry` or `pipenv` (which are mentioned in the packaging docs).

It would seem odd to provide a convenience item for something the docs don't recommend using anymore.

**Comment by di** [comment]
The guide does cover requirements files (e.g. https://packaging.python.org/guides/installing-using-pip-and-virtual-environments#using-requirements-files) and they are not deprecated.

**Comment by michaelwiles** [comment]
So then why not include lines for poetry and pipenv then? And even if requirements.txt is deprecated, don't lots of people use pip install -r requirements.txt?

I assumed because you provide a pip install line that you "support" requirements.txt...

Besides, panda, numpy, pillow, urllib3, boto3, Python-dateutil all have some kind of requirements.txt - and I'm sure if I looked more I'd find more popular python projects that do still use some kind of requirements.txt

The other flavour is that they use dependencies in this format: chardet==3.0.2 specified in the setup.py file. Which is the same format as requirements uses - and the format I'm requesting gets added.

But like I said, why not cover pipenv and poetry as well.

**Comment by sedders123** [comment]
[Nuget](https://nuget.org) the package manager for .NET has a nice UI that can show multiple install commands for different tools. 
![image](https://user-images.githubusercontent.com/1496834/83527354-97d1e900-a4df-11ea-8c47-890c7b2e703d.png) (From https://www.nuget.org/packages/Newtonsoft.Json/)

Maybe warehouse could do something similar?

**Comment by michaelwiles** [comment]
Yes! The maven repository also has that. That's why I'm asking for it.

![image](https://user-images.githubusercontent.com/1850356/83755236-874f7900-a66d-11ea-9e40-d9e2aed28cfc.png)

from https://mvnrepository.com/artifact/junit/junit/4.13
